tests/conftest.py

```diff
@@ -26,6 +26,7 @@
 TOL = 1e-3
```

Comment:
Must have blacked conftest accidentally?

Reply:
Slowly, the entire test folder will be blacked over time 😆